CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 ### Removed
 ### Fixed
 
+## [1.10.1](https://github.com/idealista/patroni_role/tree/1.0.1) (2024-06-20)
```

Comment:
There seems to be a little typo here. Could you fix it please? :)

Reply:
Corrected. Thank you.